migration/migrator/migrations/course/20180803105210_regrade_per_assignment.py

```diff
@@ -1,12 +1,16 @@
 def up(config, database, semester, course):
+    if not database.table_has_column('electronic_gradeable', 'eg_regrade_request_start_date'):
```

Comment:
You cannot edit a migration file from an old PR.   This old migrations have been marked as completed on up-to-date systems and will not be re-run.

You must make a new migration file for this PR, with the changes required for this PR.

Please read https://submitty.org/developer/migrations